helm/starboard-exporter/values.yaml

```diff
@@ -84,3 +84,10 @@ networkpolicy:
   enabled: true
 
 podAnnotations: {}
+
+customMetricsHpa:
+  enabled: false
+  minReplicas: 2
+  maxReplicas: 97
+  metricName: scrapedurationseconds
+  targetAverageValue: 10
```

Comment:
If this field has an implicit unit (e.g. seconds or minutes), that unit should be reflected in the name.

Reply:
Fixed